publisher-sdk/src/main/java/com/criteo/publisher/BidManager.java

```diff
@@ -125,6 +151,7 @@
    * @return a valid bid that may be displayed or <code>null</code> that should be ignored
    */
   @Nullable
+  // @VisibleForTesting TODO Migrate callers on #getBidForAdUnit
```

Comment:
Can you elaborate on the TODO more? JIRA ticket?

Reply:
Sure, updated